docs/core/stories.rst

```diff
@@ -102,8 +163,20 @@ it is probably better to restructure your domain and/or intents:
     * affirm OR thankyou
       - action_handle_affirmation
 
+Just like checkpoints, ``OR`` statements can be useful, but if you are using a
+lot of them, it is probably better to restructure your domain and/or intents.
 
 .. note::
+    Adding lines to your stories with many ``OR`` statements
```

Comment:
maybe include checkpoints in the "slow down training" -- should this be a warning?